qrenderdoc/Widgets/BufferFormatSpecifier.cpp

```diff
@@ -86,7 +86,9 @@ BufferFormatSpecifier::BufferFormatSpecifier(QWidget *parent)
 
   QColor base = formatText->palette().color(QPalette::Base);
 
-  QColor col = QColor::fromHslF(0.0f, 1.0f, qBound(0.1, base.lightnessF(), 0.9));
+  QColor col = IsDarkTheme()
+                   ? QColor::fromHslF(0.0f, 1.0f, qBound(0.1, 1.0f - base.lightnessF(), 0.9))
+                   : QColor::fromHslF(0.0f, 1.0f, qBound(0.1, base.lightnessF(), 0.9));
```

Comment:
Dark themes are all about light-text-on-dark-background. The bug here is that we customise the background colour without also customising the foreground colour. A better fix here would be to keep the base colour as-is but then use the `contrastingColor` helper to pick the appropriate text colour to best contrast with it (which on a dark theme will produce light text).

See e.g. `D3D11PipelineStateViewer::highlightIABind` for how it's used.